live-examples/css-examples/text/hyphens.html

```diff
@@ -0,0 +1,30 @@
+<section id="example-choice-list" class="example-choice-list" data-property="hyphens">
+<div class="example-choice">
+<pre><code id="example_one" class="language-css">hyphens: none;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code id="example_two" class="language-css">hyphens: manual;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
+<pre><code id="example_three" class="language-css">hyphens: auto;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+</section>
+
+<div id="output" class="output hidden">
+    <section id="default-example">
+        <!-- <p id="example-element">Hunting was over, and the east wind was still blowing, and a great portion of the London world was out of town taking its Easter holiday, when, on an unpleasant morning, Ferdinand Lopez travelled into the city by the Metropolitan railway from Westminster Bridge.</p> -->
```

Comment:
Commented-out HTML, this should be removed I think, unless there's a reason I'm missing?